www/src/py_exceptions.js

```diff
@@ -111,7 +111,7 @@ traceback.__getattribute__ = function(self, attr){
             if(self.$stack.length <= 1){return None}
             else{
                 return traceback.$factory(self.exc,
-                    self.exc.$stack.slice(0, self.exc.$stack.length - 2))
+                    self.$stack.slice(0, self.$stack.length - 2))
```

Comment:
I suspect that this should be using `self.$stack.length - 1` instead of 2, but I'm not certain of that yet.